packages/integrations/package.json

```diff
@@ -34,7 +34,7 @@
     "publish": "npm run build && npm run build:types"
   },
   "peerDependencies": {
-    "@relewise/client": "^2.0.0"
+    "@relewise/client": ">=1.22.0 <3.0.0"
```

Comment:
In the latest release this is how this is configured:
`"@relewise/client": "^1.22.0"`

This should allow this to be runned with any version between 1.22.0 and 3.0.0, where `3.0.0` and above is excluded

Reply:
Cool! 👏